### PR TITLE
LW6 community day blog post

### PR DESCRIPTION
**Pull request by dshukertjr**
For Friday morning PT

### CODE REVIEW

File: apps/www/_blog/2022-12-16-launch-week-6-community-day.mdx [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+We have guides on how to get started with Next.js and OneSignal here, and Flutter and OneSignal [here](https://github.com/OneSignalDevelopers/onesignal-supabase-sample-integration-supabase).
+
+William from OneSignal is here to discuss how you can easily integrate OneAignal into your Supabase app.
+
+<iframe
+  className="w-full"
+  width="560"
+  height="315"
+  src="https://www.youtube-nocookie.com/embed/In1eU6J17A8?modestbranding=1&autohide=1&showinfo=0&controls=0"
```
Comment (dshukertjr) [inline apps/www/_blog/2022-12-16-launch-week-6-community-day.mdx:48]: Will be swapped with the one uploaded on our channel.

File: apps/www/_blog/2022-12-16-launch-week-6-community-day.mdx [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+- Want to try out Supabase? Join the Hackathon and win some nice prizes.
+- Want to support us? Just a [GitHub star](https://github.com/supabase/supabase) is enough ⭐.
+
+## More Launch Week 6
+
+- [Day 1: New Supabase Docs, built with Next.js](https://supabase.com/blog/new-supabase-docs-built-with-nextjs)
+- [Day 2: Supabase Storage v2: Image resizing and Smart CDN](https://supabase.com/blog/storage-image-resizing-smart-cdn)
+- [Launch Week 6 Hackathon](https://supabase.com/blog/launch-week-6-hackathon)
+- [Who We Hire at Supabase](https://supabase.com/blog/who-we-hire)
+- [pg_crdt - an experimental CRDT extension for Postgres](https://supabase.com/blog/postgres-crdt)
```
Comment (dshukertjr) [inline apps/www/_blog/2022-12-16-launch-week-6-community-day.mdx]: @RamiroND 
Want to make sure you are happy with these.

File: apps/www/_blog/2022-12-16-launch-week-6-community-day.mdx [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+There is an awesome open source community around it who have built a [Supabase Adapter](https://next-auth.js.org/adapters/supabase) that makes it easy to use NextAuth.js with Supabase, and even enables usage of [Row Level Security (RLS) policies](https://next-auth.js.org/adapters/supabase#enabling-row-level-security-rls).
+
+In [this community spotlight video](https://youtu.be/EdYQ9fF-hz4), [Thang Huu Vu](https://twitter.com/thanghvu), the lead maintainer of NextAuth.js, gives us an overview and brief history of NextAuth, followed by a tutorial on how to set up the new Supabase Adapter. 
+
+
+<iframe
+  className="w-full"
+  width="560"
+  height="315"
+  src="https://www.youtube-nocookie.com/embed/EdYQ9fF-hz4?modestbranding=1&autohide=1&showinfo=0&controls=0"
```
Comment (dshukertjr) [inline apps/www/_blog/2022-12-16-launch-week-6-community-day.mdx:69]: Make sure the video is publically available at the time of the blog post launch.

File: apps/www/_blog/2022-12-16-launch-week-6-community-day.mdx [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+
+Over the past few years, we've had a number of people ask if we can add Full Text Search for other languages. PGroonga has been on our radar for a long time now, so when we reached out to them we were surprised to learn that _no_ Postgres platform currently supports them. Today we're fixing this by becoming the first.
+
+### PostgreSQL 15
+
+The PostgreSQL community [released](https://www.postgresql.org/docs/current/release-15.html) version 15 (stable) in October 2022. All Projects created on the Supabase platform from today will be on version 15. If you want to upgrade one of your existing Projects, check out our [Upgrading](https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects#upgrade-your-project) documentation, and if you want to see what's new, check out : XXX
+
+- Postgres 15 - [What's new in **Postgres 15?**](https://www.notion.so/What-s-new-in-Postgres-15-26edb0b5c73a4537892e7fec73188df8)
+
+### PostgREST version 11
```
Comment (steve-chavez) [inline apps/www/_blog/2022-12-16-launch-week-6-community-day.mdx:93]:
```suggestion
### PostgREST version 11 pre-release
```

File: apps/www/_blog/2022-12-16-postgrest-11-prerelease.mdx [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+## Order by related tables
+
+It's also a common use case to order a table by a related table column. For example, suppose you'd like to order `films` based on the `technical_specs.duration` column.
+
+You can now do it like so:
+
+<small>From HTTP:</small>
+
+```http
+GET /films?select=*,...technical_specs(camera,laboratory,duration)&order=technical_specs(duration)
```
Comment (steve-chavez) [inline apps/www/_blog/2022-12-16-postgrest-11-prerelease.mdx:105]:
```suggestion
GET /films?select=title,...technical_specs(duration)&order=technical_specs(duration)
```

File: apps/www/_blog/2022-12-16-postgrest-11-prerelease.mdx [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+```
+
+But we'd like to present a “flattened” result to the frontend, without the `technical_specs` object. For this we could create a new database view or function that shapes the json the way we want, but creating extra database objects is not always convenient.
+
+Using the new “spread” operator(syntax borrowed from [JS](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax)), we can expand a related table columns and remove the nested object.
+
+<small>From HTTP:</small>
+
+```jsx
+GET /films?select=*,...technical_specs(camera,laboratory,duration)
```
Comment (steve-chavez) [inline apps/www/_blog/2022-12-16-postgrest-11-prerelease.mdx:66]:
```suggestion
GET /films?select=title,...technical_specs(camera,laboratory,duration)
```

File: apps/www/_blog/2022-12-16-postgrest-11-prerelease.mdx [hunk 7]
```diff
@@ -0,0 +1,226 @@
...
+      {"rank": 1, "name": "Palme d'Or"},
+      {"rank": 1, "name": "BAFTA Film Award"},
+      {"..."}
+    ]
+  },
+  "..."
+]
+```
+
+This was already possible with the `!inner` modifier([introduced on PostgREST 9](https://supabase.com/blog/postgrest-9#resource-embedding-with-inner-joins_)
```
Comment (steve-chavez) [inline apps/www/_blog/2022-12-16-postgrest-11-prerelease.mdx:215]:
```suggestion
This was already possible with the `!inner` modifier([introduced on PostgREST 9](https://supabase.com/blog/postgrest-9#resource-embedding-with-inner-joins))
```

File: apps/www/_blog/2022-12-16-launch-week-6-community-day.mdx [hunk 8]
```diff
@@ -0,0 +1,240 @@
...
+
+Watch John from the FlutterFlow team demonstrate how you can [get started with FlutterFlow and Supabase](https://youtu.be/TmViZsWEBY0).
+
+### OneSignal
+
+<iframe
+  className="w-full"
+  width="560"
+  height="315"
+  src="https://www.youtube-nocookie.com/embed/In1eU6J17A8?modestbranding=1&autohide=1&showinfo=0&controls=0"
```
Comment (dshukertjr) [inline apps/www/_blog/2022-12-16-launch-week-6-community-day.mdx:47]:
```suggestion
  src="https://www.youtube-nocookie.com/embed/mw0DLwItue4?modestbranding=1&autohide=1&showinfo=0&controls=0"
```

File: apps/www/_blog/2022-12-16-launch-week-6-community-day.mdx [hunk 9]
```diff
@@ -0,0 +1,240 @@
...
+
+Read [what's new in **Postgres 15?**](https://www.notion.so/What-s-new-in-Postgres-15-26edb0b5c73a4537892e7fec73188df8)
+
+### PostgREST version 11 pre-release
+
+![Postgrest 11](/images/blog/lw6-community/postgrest.png)
+
+PostgREST 11 is not wrapped up yet but you can try a pre-release locally with the Supabase CLI. Some of the new features include: spreading related tables, related orders and anti-joins. Read on more about these on:
+
+[Learn more about PostgREST 11](https://supabase.com/blog/launch-week-6-hackathon/postgrest-11-pre)
```
Comment (dshukertjr) [inline apps/www/_blog/2022-12-16-launch-week-6-community-day.mdx:107]:
```suggestion
[Learn more about PostgREST 11](https://supabase.com/blog/postgrest-11-prerelease)
```

File: apps/www/_blog/2022-12-16-launch-week-6-community-day.mdx [hunk 10]
```diff
@@ -0,0 +1,240 @@
...
+
+While we've been working hard on new features, the community has been developing a healthy [integrations marketplace](https://supabase.com/docs/guides/integrations/integrations). Since Supabase is just Postgres, any tool which works with Postgres works with Supabase (no additional changes required). Today we're adding a few major players to the roster:
+
+### FlutterFlow
+
+<iframe
+  className="w-full"
+  width="560"
+  height="315"
+  src="https://www.youtube-nocookie.com/embed/TmViZsWEBY0?modestbranding=1&autohide=1&showinfo=0&controls=0"
```
Comment (dshukertjr) [inline apps/www/_blog/2022-12-16-launch-week-6-community-day.mdx:26]:
```suggestion
  src="https://www.youtube-nocookie.com/embed/hw9Q-NjASbU?modestbranding=1&autohide=1&showinfo=0&controls=0"
```

**Review by awalias — APPROVED** [review]

File: apps/www/_blog/2022-12-16-launch-week-6-community-day.mdx [hunk 11]
```diff
@@ -0,0 +1,240 @@
...
+  allowfullscreen
+></iframe>
+
+[NextAuth.js](https://next-auth.js.org/) is an open source authentication solution for [Next.js](https://nextjs.org/). It allows you to bring your own Database so that you own your user data.
+
+The amazing NextAuth community who have built a [Supabase Adapter](https://next-auth.js.org/adapters/supabase) that makes it easy to use NextAuth.js with Supabase, and enables the use of [Row Level Security (RLS) policies](https://next-auth.js.org/adapters/supabase#enabling-row-level-security-rls).
+
+In [this community spotlight video](https://youtube.com/EdYQ9fF-hz4), [Thang Huu Vu](https://twitter.com/thanghvu), the lead maintainer of NextAuth.js, gives an overview and brief history of NextAuth, followed by a tutorial on how to set up the new Supabase Adapter.
+
+## New Tools and Updates
```
Comment (awalias) [inline apps/www/_blog/2022-12-16-launch-week-6-community-day.mdx:81]:
```suggestion
## Postgres Ecosystem
```

**Pull request merged**